### ISSUE TITLE
TextEdit: setting scroll_vertical does not always work properly

### ISSUE THREAD

**Issue by BevanFindlay** [godotengine/godot#46606]
**Godot version:**
v3.2.3.stable.official

**OS/device including version:**
Windows 10 20H2

**Issue description:**
I have a TextEdit that is set to readonly, used to display help texts in a panel that can be hidden (visible = false).  The help texts are just strings that get applied to the TextEdit's .text property.  These strings have wrapped lines, but I am not sure if that is relevant.
If the control is invisible, setting scroll_vertical does not work.  It will only update the scroll position if the control is visible.  As far as I can tell, this is not documented (i.e. that you cannot set scroll_vertical on a hidden control), but probably should be.
However, even if I set scroll_vertical after the TextEdit is set to be visible, it sometimes does not work (in my case, when I re-open the same help text - i.e. do not change the text).  I managed to work around it by adding yield(get_tree(),"idle_frame") before I set the scroll position, but it's not clear from any documentation that this is what is required, and ideally it should not be needed.

So the issue is in two parts: (1) a TextEdit must be visible or setting scroll_vertical has no effect (sometimes meaning that you need to wait a frame before setting it), and (2) none of this is documented.

**Steps to reproduce:**
As above.

**Minimal reproduction project:**
Let me know if this is needed.

**Comment by Calinou** [comment]
> Let me know if this is needed.

Indeed, please upload a [minimal reproduction project](https://github.com/godotengine/godot/blob/master/CONTRIBUTING.md#provide-a-simple-example-project) to make this easier to troubleshoot.

**Comment by BevanFindlay** [comment]
As requested.  If you scroll down then click the _Set scroll to top_ button, it scrolls up fine.  If you scroll down, _Toggle visibility_, and then click _Set scroll to top_, it is not scrolled up when it is made visible again.

[TextEdit test project.zip](https://github.com/godotengine/godot/files/6119757/TextEdit.test.project.zip)
